cheroot/server.py

```diff
@@ -652,7 +654,10 @@ def read_request_headers(self):
 
         # then all the http headers
         try:
-            read_headers(self.rfile, self.inheaders)
+            # Configurable using `"server.drop_underscore_headers": True`:
+#             drop_underscores = getattr(self.server.server_adapter, 'drop_underscore_headers', False)
```

Comment:
No this should not be merged. I just proposed two variants for discussion. I'll add a comment to the issue #17